packages/forklift-console-plugin/src/plans/create/steps/GeneralInformationForm.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FC } from 'react';
+import { Controller, useFormContext } from 'react-hook-form';
+import { FormErrorHelperText } from 'src/components/FormErrorHelperText';
```

Comment:
could you also please ontorduce new path in tsconfig and webpack for @component ?